src/main/java/com/dabsquared/gitlabjenkins/connection/GitLabConnectionProperty.java

```diff
@@ -167,18 +168,20 @@ public FormValidation doTestConnection(
                                 "",
                                 gitLabConnectionTested.getUrl(),
                                 jobCredentialId,
-                                gitLabConnectionTested.getClientBuilderId(),
                                 true,
                                 gitLabConnectionTested.getConnectionTimeout(),
                                 gitLabConnectionTested.getReadTimeout())
-                        .getClient(item, jobCredentialId)
+                        .getGitLabApi(item, jobCredentialId)
+                        .getUserApi()
                         .getCurrentUser();
                 return FormValidation.ok(Messages.connection_success());
             } catch (WebApplicationException e) {
                 return FormValidation.error(Messages.connection_error(e.getMessage()));
             } catch (ProcessingException e) {
                 return FormValidation.error(
                         Messages.connection_error(e.getCause().getMessage()));
```

Comment:
Ditto regarding catching `ProcessingException` being unnecessary since this could only be thrown by `RESTEasy`